### PR TITLE
Remove Duplicates (Ungoogled Chromium & Telegram X)

### PR DESCRIPTION
**Pull request by IAmRuMaks**

### CODE REVIEW

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Please correct me if I’m wrong, but I believe the name ungoogled-chromium is written in lowercase letters and hyphenated. An example: https://en.wikipedia.org/wiki/Ungoogled-chromium.

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -301,7 +301,7 @@
     <icon drawable="@drawable/twitch" package="tv.twitchmod.android.app" name="TwitchMod" />
     <icon drawable="@drawable/twitter" package="com.twitter.android" name="Twitter" />
     <icon drawable="@drawable/twitter" package="com.twitter.android.lite" name="Twitter" />
-    <icon drawable="@drawable/ungoogled_chromium" package="org.ungoogled.chromium.stable" name="ungoogled-chromium" />
+    <icon drawable="@drawable/google_chrome" package="org.ungoogled.chromium.stable" name="Ungoogled Chromium" />
```
Comment (patrykmichalik) [inline app/src/main/res/xml/grayscale_icon_map.xml:304]:
```suggestion
    <icon drawable="@drawable/google_chrome" package="org.ungoogled.chromium.stable" name="ungoogled-chromium" />
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
It looks like I didn’t include an interactive change request in my previous review. Sorry! Here it is.

**Comment by IAmRuMaks** [comment]
No, it's called "Ungoogled Chromium":
Screenshotl

**Pull request merged**